egs/librispeech/s5/local/nnet3/tuning/run_tdnn_1b.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+# 1b is as 1a but uses xconfigs.
+
+# this is the standard "tdnn" system, built in nnet3; it's what we use to
```

Comment:
If this is a new script, can you please display the results with local/chain/compare_wer.sh?  Doesn't have to be a comparison *with* something.  Just its output-- and the output of steps/info/chain_dir_info.pl.  And can you please also run a more modern TDNN configuration, something with tdnnf-layer?  Doesn't have to hold up this PR though.

Reply:
OK. I'll make these changes in another PR.